lxd/instance/instance_utils.go

```diff
@@ -1050,6 +1050,11 @@ func NextSnapshotName(s *state.State, inst Instance, defaultPattern string) (str
 	return pattern, nil
 }
 
+// CreateTemporaryName creates the temporary name for moving instances.
+func CreateTemporaryName(instUUID string) string {
```

Comment:
Does this need to be exported?

Can we just call this `TemporaryName` or `GetTemporaryName` as it doesn't really create anything.

Reply:
True, no need for it to be exported. Changed.